django/contrib/admin/templates/admin/auth/user/change_password.html

```diff
@@ -30,6 +34,28 @@
 
 <fieldset class="module aligned">
 
+<div class="form-row">
+  {{ form.usable_password.errors }}
+  <div class="flex-container checkbox-row">{{ form.usable_password }}
+    <label class="vCheckboxLabel" for="id_usable_password">{{ form.usable_password.label }}</label>
+  </div>
+  {% if form.usable_password.help_text %}
+  <div class="help"{% if form.usable_password.id_for_label %} id="{{ form.usable_password.id_for_label }}_helptext">{% endif %}
```

Comment:
">" should be outside of `{% if ... %}...{% endif %}`/

Reply:
Indeed! 🤦‍♂️